pastas/stressmodels.py

```diff
@@ -762,9 +764,14 @@ def set_init_parameters(self):
             np.max(self.distances) ** 2
         self.parameters.loc[self.name + "_b", "pmin"] /= \
             np.max(self.distances) ** 2
-        # set initial value with mean distance
+        # set initial value b with mean distance,
+        # filling in mean distance as r will yield 1 as initial value
```

Comment:
I think this should be done in HantushWellModel.get_init_parameters(), just pass on r. That also makes it easier to add similar response functions that use a "distance" in the future no?

Reply:
Good idea, I'll change it with the next commit